docs/index.md

```diff
@@ -6,55 +6,56 @@ actionText: Get started
 actionLink: /getting-started/
 features:
 - title: "Customizable"
-  details: "LDK was designed from the ground up to be easily customized to your application needs: persistence, networking, chain source, routing, key management, wallet, you name it."
+  details: "Designed from the ground up to be easily customized to your application needs: persistence, networking, chain source, routing, key management, wallet, you name it."
   image: "customizable"
 - title: "Focus on what matters"
-  details: "LDK lets you focus on your user experience and we'll handle all the low-level lightning logic."
+  details: "All of the low-level lightning logic is handled by us, so you can focus on crafting custom-tailored user experiences."
   image: "focus"
-- title: "Mobile first"
-  details: "As lightweight as you need it to be and with language bindings suitable for iOS or Android -- it may be written in secure Rust, but you'll never have to touch Rust code."
+- title: "High performance & compact"
+  details: "As lightweight as you need it to be and optimized to run on all modern-day embedded devices such as mobile phones, IoT devices, PoS terminals and more."
   image: "mobile"
 ---
 
 <div class="intro">
 <h2>Why LDK?</h2>
-<p>Leading Bitcoin open source projects love LDK …</p>
+<p>Leading Bitcoin open source projects love LDK</p>
 </div>
 
 <div class="features">
 <div class="feature">
-<h3>Cross-Language Bindings</h3>
+<h3>Multi-Language Support</h3>
 
-As the application is an SDK and not a standalone binary opinionated about its execution environment, we would provide FFI bindings into whichever languages developers happen to build Lightning applications in, be they C, Swift, or Python. You can open the same wallet in multiple environments (e.g. in a web browser, on a phone, on a desktop, etc)!
```

Comment:
Worth mentioning the second part - "You can open the same wallet in multiple environments (e.g. in a web browser, on a phone, on a desktop, etc)!"? ie you can open the same wallet in software written in different languages.